lib/utils/compose.coffee

```diff
@@ -41,6 +41,12 @@ exports.appendOptions = (opts) ->
 			description: 'Display full log output'
 			boolean: true
 		},
+		{
+			signature: 'registry-secrets',
+			alias: 'R',
+			parameter: 'secrets.yml|.json',
+			description: 'Path to a YAML or JSON file with passwords for a private Docker registry',
+		},
```

Comment:
FYI, this option is shared by the `build` and `deploy` commands. In case it occurs to you, I realise that the `registry-secrets` spelling with a dash does not match the camelCase spelling of the other `build` and `deploy` commands, but it is the spelling used by the `push` command and there is also the goal of having the same options across all relevant CLI commands. There've been separate discussions about the homogenisation of CLI command options and it is part of the longer-term CLI redesign plans. It would be nice to accept both variants (reduces user friction, I suppose), but it's something for a future PR.